### ISSUE TITLE
implementation of PRM

### ISSUE THREAD

**Issue by rishishian** [caiyuanhao1998/RSN#5]
It seems that you forget to check implementation of PRM module.
In [[your implementation of PRM class]](https://github.com/caiyuanhao1998/RSN/blob/master/exps/4XRSN18.coco/network.py) ,  

- You define sigmoid2 in init but call self.sigmoid in forward func. 
- It seems that there is no torch.nn.GlobalAvgPool2d() in pytorch. 
- You use out1+out2+out3,  however, in your paper, there should be a multiplication between middel path(out2) and bottom path(out3)

I know this is just a demo implementation, but it will be better if you correct it.

Finally, MegEngine has been released, but there's no implementation of RSN or its former work(CPN, MSPN). May I ask when is the right time to release? I have followed your work for some time, thanks !

**Comment by caiyuanhao1998** [comment]
Thanks a lot for pointing out the mistack, I've corrected it. I am busy with my graduation project now. I can't tell you the exact time.

**Comment by rishishian** [comment]
> Thanks a lot for pointing out the mistack, I've corrected it. I am busy with my graduation project now. I can't tell you the exact time.

Okay~ I've closed this issue. I think this paper&repo is good enough to serve as graduation project, why not just use it ? hahaha
